packages/mask/src/plugins/ENS/SNSAdaptor/ENSEmptyContent.tsx

```diff
@@ -0,0 +1,20 @@
+import useStyles from './useStyles'
+import { useI18N } from '../locales'
+import { Icons } from '@masknet/icons'
+import { Typography, Box } from '@mui/material'
+
+export function ENSEmptyContent() {
```

Comment:
Components in ENS plugins are already in the ENS context. I think we could omit it for the sake of simplicity.